test/e2e/util.go

```diff
@@ -333,7 +341,7 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	kubectlArgs := append(defaultArgs, args...)
 
 	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command("kubectl", kubectlArgs...)
+	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
```

Comment:
I'm a little confused. `hack/ginkgo-e2e.sh` was setting `PATH` to include the directory containing `e2e`, which as best as I understand, will result in the version of `kubectl` from the release being used. (One difference is that kubectl.sh tacks on some extra args for gke, I guess.)